api/handlers_e2e_test.go

```diff
@@ -78,7 +78,7 @@ func setupIdb(t *testing.T, genesis bookkeeping.Genesis) (*postgres.IndexerDb, f
 	log, _ := test2.NewNullLogger()
 	l, err := test.MakeTestLedger(log)
 	require.NoError(t, err)
-	proc, err := blockprocessor.MakeProcessorWithLedger(l, db.AddBlock)
+	proc, err := blockprocessor.MakeProcessorWithLedger(log, l, db.AddBlock)
```

Comment:
in some files, this parameter is named logger or l. can we update them to be the same in all places?

Reply:
Done, I went with logger. I think thats the name we use most often.